src/components/PrincipalSection/index.tsx

```diff
@@ -1,10 +1,52 @@
+import Diablo4 from '~/../../public/assets/logos/diablo-4.png'
+import HearthStone from '~/../../public/assets/logos/hearthstone.png'
+import WoW from '~/../../public/assets/logos/world-of-warcraft.png'
+import Diablo1 from '~/../../public/assets/logos/diablo-1.png'
+import StarCraft2 from '~/../../public/assets/logos/star-craft-2.png'
+import Diablo4XL from '~/../../public/assets/logos/diablo-4xl.png'
+import ThumbDiablo4 from '~/../../public/assets/banner/thumb-diablo4.png'
+
 import { Navbar } from 'components'
 
 const PrincipalSection = () => {
   return (
     <div className="min-h-[738px] bg-1 ">
```

Comment:
Yes, the old name was bg-banner-1, but I renamed for 1 because the bg It's self explanatory I was thinking.

Reply:
> because the bg It's self explanatory I thought

Not for anyone new to the codebase. I strongly believe replacing this class with inline rules (you should only need `background-url`) would make it much easier to understand what is actually happening.